app/detekt-baseline.xml

```diff
@@ -0,0 +1,361 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I assume we will fix some of these in the future? (same question for the other modules)

Reply:
I hope so.
At least I do not want to do it right now, as it will take a lot of time.